server/mcp_server_askecho/README.md

```diff
@@ -21,15 +21,77 @@ v0.1.0
 
 本 MCP Server 产品提供以下 Tools (工具/能力):
 
-### Tool 1: chat_completion
+### Tool 1: chat_completion_official
 
 #### 类型
 
 saas
 
 #### 详细描述
 
-根据用户输入问题，提供基于联网搜索的大模型总结后回复内容
+根据用户输入问题，提供基于联网搜索的大模型总结后回复内容（推荐使用）
+
+#### 调试所需的输入参数:
+
+输入：
+
+```json
+{
+  "inputSchema": {
+    "type": "object",
+    "required": [
+      "messages"
+    ],
+    "properties": {
+      "messages": {
+        "description": "对话消息列表",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "required": [
+            "role",
+            "content"
+          ],
+          "properties": {
+            "role": {
+              "description": "消息角色类型",
+              "type": "string"
+            },
+            "content": {
+              "description": "消息内容",
+              "type": "string"
+            }
+          }
+        }
+      },
+      "user_id": {
+        "description": "可选，用户标识符，用于标识用户身份，以支持新闻相关工具的去重逻辑",
+        "type": "string"
+      }
+    }
+  },
+  "name": "chat_completion_official",
+  "description": "联网问答智能体会话-正式版本"
+}
+```
+
+输出：
+
+- 大模型基于联网搜索给出的总结回复
+
+#### 最容易被唤起的 Prompt示例
+
+今天的热点新闻
+
+### Tool 2: chat_completion
```

Comment:
replace chat_completion in place, check if ASKECHO_CHAT_COMPLETION_OFFICIAL_USER_ID is available to determine backend implemetation.